### DISCUSSION TITLE
docsify的中文文档。docsify's Chinese documentation

### DISCUSSION THREAD

**Discussion by sy-records** [docsifyjs/docsify#1806] [Show and tell]
由于近期中国境内对于`cdn.jsdelivr.net`产生了DNS污染，导致无法正常访问，所以临时将`cdn.jsdelivr.net`的资源替换为了`unpkg.com` 

> ~或者可以使用 `fastly.jsdelivr.net` 替换 `cdn.jsdelivr.net`~

https://github.com/sy-records/docsify/commit/c894f533ad9a81c7e51da50b4d6cc463b54e4452

对于其他地域的用户并没有影响，正常使用 https://docsify.js.org。

中国用户可以暂时使用 https://zh.docsifyjs.org/#/zh-cn/ 来正常访问文档。

----

Due to the recent DNS pollution of `cdn.jsdelivr.net` in China, the resource `cdn.jsdelivr.net` is temporarily replaced with `unpkg.com`. 

https://github.com/sy-records/docsify/commit/c894f533ad9a81c7e51da50b4d6cc463b54e4452

For users in other countries and regions there is no impact, use https://docsify.js.org as normal.

Chinese users can temporarily use https://zh.docsifyjs.org/#/zh-cn/ to access the documentation.

**Comment by S-N-O-R-L-A-X** [comment]
Thanks a lot. But using `fastly.jsdelivr.net` does not work for me. Using cdn from bytedance works. [cdn bytedance](https://cdn.bytedance.com/?query=docsify&version=4.12.2)

**Reply by sy-records** [reply]
yep, need change to other cdn.
